sn_networking/src/lib.rs

```diff
@@ -656,6 +657,34 @@ impl Network {
                         }
                     }
                 }
+                // Accumulate scratchpad
+                else if kind == &RecordKind::Scratchpad {
```

Comment:
would prefer this to be changed to `match` pattern, so that any future ones won't got `skipped through` accidently